src/init/omp.lua

```diff
@@ -56,7 +57,14 @@ function p:filter(prompt)
     return get_posh_prompt(false)
 end
 function p:rightfilter(prompt)
-    return get_posh_prompt(true), false
+    if tip_word == nil then
+        return get_posh_prompt(true), false
+    end
+    local prompt_exe = string.format('::OMP:: --config="::CONFIG::" --command="%s"', tip_word)
+    tooltip = io.popen(prompt_exe):read("*a")
+    if tooltip ~= nil and tooltip ~= "" then
```

Comment:
Probably it should always `return tooltip, false`.  Otherwise if there's no rprompt string then it will allow other prompt filter functions to run, and something else's rprompt could get used by mistake.

Reply:
On second thought it needs to return a string rather than nil, otherwise it won't override any prompt filters that might run before it.  So the return line should become `return (tooltip or ""), false`.

Reply:
resolved it by using `return (tooltip or get_posh_prompt(true)), false`